src/Config/Manager.php

```diff
@@ -73,19 +73,27 @@ function ($carry, $sync_id) {
       []
     );
 
-    // Filter out any synchronization that do not match the entity type ID
-    // filter, if given.
+    // Filter out any synchronization that do not match the entity type ID and
+    // bundle filters, if given.
     $syncs = array_filter(
       $syncs,
       function ($sync) use ($filters) {
         if (!$filters['local_entity']['type_id']) {
           return TRUE;
         }
 
+        if (!$filters['local_entity']['bundle']) {
+          return TRUE;
+        }
+
```

Comment:
**CHANGE REQUEST**

You need to add this to the default options as `NULL` at the beginning of the method. And to update the interface documentation.